pkg/service/utgen/assets/test_generation.toml

```diff
@@ -180,6 +335,11 @@ Use block scalar('|') to format each YAML output.
 {% endif %}
 
 
+In Python, Since  async def functions are not natively supported for testing and can be skipped, please add the decorator @pytest.mark.asyncio to the test functions that are async def functions, also if you are using the decorator, please import pytest and pytest-asyncio as well.
+In Python, use absolute imports instead of relative imports to avoid issues with module resolution.
+
+In Golang, please import testing library and use the testing.T object to run the tests.
+
```

Comment:
any reason of keep this here in this format?

hopefully we don't give it a lot of texts, lets try to keep it crisp and concise, as a lot of instruction might not be the actual help

Reply:
no not specific reason, could you please tell me where can i keep this, wasnt able to figure out